components/modal.jsx

```diff
@@ -1,10 +1,9 @@
-var React = require('react');
-var ReactAddons = require('react/addons');
+var React = require('react/addons');
```

Comment:
Is this mean we will have access to the normal `React` object plus `addons` mixins at the same time?